bridge/test/OpenSeaMint_test.js

```diff
@@ -0,0 +1,61 @@
+/* libraries used */
+const { toBN } = require('web3-utils')
+const utils = require('./utils');
+
+/* Contracts in this test */
+const OpenSea721 = artifacts.require('./OpenSea721.sol');
+
+contract.only('Interaction with OpenSea721', (accounts) => {
+  const NAME = 'ERC-271 Test Contract';
+  const SYMBOL = 'ERC71Test';
+
+  const owner = accounts[0];
+  const otherAccount = accounts[1];
+  const userA = accounts[2];
+
+
+  let instance;
+  // Keep track of token ids as we progress through the tests, rather than
+  // hardcoding numbers that we will have to change if we add/move tests.
+  // For example if test A assumes that it will create token ID 1 and test B
+  // assumes that it will create token 2, changing test A later so that it
+  // creates another token will break this as test B will now create token ID 3.
+  // Doing this avoids this scenario.
+  let tokenId = 0;
+
+  before(async () => {
+    instance = await OpenSea721.new(NAME, SYMBOL);
+  });
```

Comment:
Question, do you do this to not use beforeEach and deploy a new contract at each test?